components/homme/cmake/HommeMacros.cmake

```diff
@@ -110,7 +110,7 @@ macro(createTestExec execName execType macroNP macroNC
   ADD_EXECUTABLE(${execName} ${EXEC_SOURCES})
   SET_TARGET_PROPERTIES(${execName} PROPERTIES LINKER_LANGUAGE Fortran)
 
-  IF (${CXXLIB_SUPPORTED})
+  IF (CXXLIB_SUPPORTED_CACHE)
```

Comment:
Is this fixing a separate issue? That's fine, but I'm just checking this bit is orthogonal to the PIO part to be sure I understand what's going on.

Reply:
It is separate, this check did not work (we prob. never ran on machine without this flag supported?).